apps/namadillo/src/atoms/fees/atoms.ts

```diff
@@ -51,6 +52,18 @@ export const gasEstimateFamily = atomFamily(
             counter("RevealPk"),
             counter("Redelegate"),
           ]);
+
+          // TODO: we need to improve this estimate API. Currently, gasEstimate.min returns
+          // the minimum gas limit ever used for a TX, and avg is failing in most of the transactions.
+          // So we estabilish the average value between gasEstimate.avg and gasEstimate.max as the min value.
+          // The other values derive from this one.
+          const newMin = Math.ceil((gasEstimate.avg + gasEstimate.max) / 2);
```

Comment:
I understand the reason, but I'm afraid of this math creates an infinite loop of gas growing infinitely. We need to discuss this with other folks, but I'm good for now on campfire

For me, this endpoint should be the return value of a dry-run, then we apply a const like the 1.25 and 1.5 you did